app/views/partials/_formPost.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="form">
+    <div class="form-head">Add new Post</div>
+    <%= form_with model: @post, url: '/posts/create', scope: :post do |form| %>
+      <%= form.text_field :title, placeholder: 'Enter post title', required: true %>
+      <%= form.text_area :text, placeholder: 'Enter post text', required: true %>
+      <%= form.submit "Submit post" %>
+    <% end %>
+</div>
```

Comment:
- [ ]  I see you deleted the update_posts_counter method. That's why ı am getting errors when trying to create a post. Please fix it.

![image](https://user-images.githubusercontent.com/38635158/167009990-f6032868-6028-4b5d-8b18-ccee11b1669a.png)